Website/Providers/DataProviders/SqlDataProvider/07.04.01.SqlDataProvider

```diff
@@ -117,6 +117,49 @@ AS
 
 GO
 
+-- DNN-6702: fix an issue with shared modules in Site Groups
+-- ---------------------------------------------------------
+IF EXISTS(SELECT * FROM sys.indexes WHERE name = 'IX_{objectQualifier}ModulePermission' AND object_id = OBJECT_ID('{objectQualifier}ModulePermission'))
+	ALTER TABLE	{databaseOwner}[{objectQualifier}ModulePermission] DROP CONSTRAINT [IX_{objectQualifier}ModulePermission] 
+GO
```

Comment:
that's right, but I've ssen too many sites, where Upgrades had issues and it doesn't cost any effort. But it is up to you.